packages/core/src/main/utils/resolve-system-proxy/resolve-system-proxy-window.injectable.ts

```diff
@@ -4,13 +4,21 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { BrowserWindow } from "electron";
+import electronAppInjectable from "../../electron-app/electron-app.injectable";
 
 const resolveSystemProxyWindowInjectable = getInjectable({
   id: "resolve-system-proxy-window",
-  instantiate: () => {
-    const window = new BrowserWindow({ show: false });
+  instantiate: async (di) => {
+    const app = di.inject(electronAppInjectable);
 
-    window.hide();
+    await app.whenReady();
```

Comment:
Cleaner way to do this:
`const app = await di.inject(readyElectronAppInjectable);`